src/postgraphql/withPostGraphQLContext.js

```diff
@@ -0,0 +1,45 @@
+import { $$pgClient } from '../postgres/inventory/pgClientFromContext'
+import debugPgClient from './http/debugPgClient'
+import setupPgClientTransaction from './setupPgClientTransaction'
+
+/*
+
+This is intended to be called similar to this:
+
+    const result = await withPostGraphQLContext(options, { pgPool, jwtToken }, async context => (
+      await graphql(
+        schema,
+        query,
+        null,
+        {  ...context },
+        variables,
+        operationName,
+      )
+    ))
+
+*/
+
+export default async function withPostGraphQLContext(options, {pgPool, jwtToken}, functionToRun) {
```

Comment:
Could we add some types to this? Also, could we merge `options` and the `{ pgPool, jwtToken }` object into one? This is generally the pattern we’ve gone with for other functions.